app/states/unified_database.py

```
@@ -0,0 +1,1208 @@
+                    }
+                    unlinked_transactions.append(tx_person)
+            
+            return unlinked_transactions
+
+
+
+# Global database manager instance (PostgreSQL)
+# Only created if PostgreSQL connection is available
+try:
+    db_manager = UnifiedDatabaseManager()
+except (RuntimeError, Exception) as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)